backend/src/repositories/reportRepository.js

```
@@ -0,0 +1,59 @@
+    );
+
+    return {
+      totalVolunteers: parseInt(stats.totalVolunteers || 0, 10),
+      totalItems: parseInt(stats.totalItems || 0, 10),
+      totalOrganizations: parseInt(orgStats.orgCount || 0, 10),
+      totalTeams: parseInt(teamStats.teamCount || 0, 10),
+      organizations,
+      teams,
+    };
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)